source/ddbus/package.d

```diff
@@ -1,3 +1,5 @@
+module ddbus;
```

Comment:
I think this makes the code more clear and should definitely be done, but not in the scope of this PR